src/bosh-director/lib/bosh/director/addon/addon_parser.rb

```diff
@@ -0,0 +1,37 @@
+module Bosh::Director
+  module Addon
+    class AddonParser
```

Comment:
Same thing as `Filter` - make sense to rename to `Bosh::Director::Addon::Parser`?